pkg/conversion/error.go

```diff
@@ -54,6 +54,10 @@ type missingKindErr struct {
 	data string
 }
 
+func NewMissingKindErr(data string) error {
+	return &missingKindErr{data}
+}
+
 func (k *missingKindErr) Error() string {
```

Comment:
/grumpycat error types should be public for introspection. You don't need to fix in this PR though.